cmd/lakectl/cmd/root.go

```diff
@@ -140,6 +147,10 @@ const (
 	allowEmptyMsgFlagName = "allow-empty-message"
 	fmtErrEmptyMsg        = `commit with no message without specifying the "--allow-empty-message" flag`
 	metaFlagName          = "meta"
+
+	defaultMaxRetries       = 3
+	defaultMaxRetryInterval = 1 * time.Second
```

Comment:
Too low IMO
```suggestion
	defaultMaxRetryInterval = 5 * time.Second
```

Reply:
I used a different value, but raised the max interval.